### ISSUE TITLE
Screen Reader: Settings-About: The links are announced without a link role

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74885]
## Action Performed
Prerequisite:
- the user is logged in

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/about
2. Navigate using TAB to 'App download links', press enter to activate
3. On the side panel pop up menu, navigate using TAB to each of the download links (Android, iOS, macOS)
4. Observe the announcements

## Expected Result
Each of the app download links should be announced with an lnk role.

## Actual Result
When users navigate to the list of app download links, they are announced without the role of link, and only the visual text for each element.
-"Android"
-"iOS"
-"macOS"

HTML:
<div class="css-g5y9jx" data-component="AnimatedComponent" data-element="Component" data-source-file="createAnimatedComponent.js" style="opacity: 1;"><div aria-label="Android" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-1mmae3n r-u9wvl5 r-13qz1uu" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci" style="width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xmlns:xlink="[http://www.w3.org/1999/xlink"](http://www.w3.org/1999/xlink%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><defs><path id="android_svg__a" d="M1 5h17.8v10H1z"></path></defs><clipPath id="android_svg__b"><use xlink:href="#android_svg__a" style="overflow: visible;"></use></clipPath><path d="M14 12.5c-.4 0-.7-.3-.7-.7s.3-.7.7-.7.7.3.7.7c0 .3-.3.7-.7.7m-8.2 0c-.4 0-.7-.3-.7-.7s.3-.7.7-.7.7.3.7.7c0 .3-.3.7-.7.7M14.2 8l1.5-2.6c.1-.1 0-.3-.1-.4s-.3 0-.4.1l-1.5 2.6c-1.1-.5-2.4-.8-3.8-.8s-2.7.3-3.8.8L4.6 5.2C4.5 5 4.3 5 4.2 5c-.2.1-.2.3-.1.5L5.5 8C3 9.4 1.3 12 1 15h17.8c-.3-3-2-5.6-4.6-7" style="clip-path: url(&quot;#android_svg__b&quot;);"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(0, 46, 34); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex-shrink: 1; margin-left: 12px; white-space: pre; direction: ltr; max-width: 100%; line-height: 20px;">Android</div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"><div class="css-g5y9jx r-105ug2t" style="justify-content: center; align-items: center; width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><path d="M13.9 4 8.4 9.4c-.6.6-.6 1.5 0 2.1s1.5.6 2.1 0L16 6.1v2.4c0 .8.7 1.5 1.5 1.5S19 9.3 19 8.5V1h-7.5c-.8 0-1.5.7-1.5 1.5S10.7 4 11.5 4z"></path><path d="M1 9.5C1 7.6 2.6 6 4.5 6H7c.8 0 1.5.7 1.5 1.5S7.8 9 7 9H4.5c-.3 0-.5.2-.5.5v6c0 .3.2.5.5.5h6c.3 0 .5-.2.5-.5v-2.4c0-.8.7-1.5 1.5-1.5s1.5.7 1.5 1.5v2.4c0 1.9-1.6 3.5-3.5 3.5h-6C2.6 19 1 17.4 1 15.5z"></path></svg></div></div></div></div></div></div></div>

## Area issue was found
Settings-About

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Remove role="menuitem" and implement the controls as native <a> elements with a valid href attribute.
https://www.w3.org/WAI/ARIA/apg/patterns/link/

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/64910a6c-b471-449a-b6bc-ebd59fa14b91

<img width="1915" height="1003" alt="Image" src="https://github.com/user-attachments/assets/53b693f0-c182-4f7d-93b0-9ddcc530d1a4" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995297
**Issue reported by:** NA
**Slack conversation:** NA
